src-vue/package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@element-plus/icons-vue": "^2.0.9",
     "element-plus": "^2.2.17",
+    "tauri-plugin-store-api": "github:tauri-apps/tauri-plugin-store#dev",
```

Comment:
The #dev means npm should use `dev` branch?

Reply:
Yup, there's no tagged release yet unfortunately: https://github.com/tauri-apps/tauri-plugin-store

I basically just did `npm install https://github.com/tauri-apps/tauri-plugin-store\#dev`

Reply:
Aren't there more stable branches to rely on?